messages/context.json

```
@@ -0,0 +1,14 @@
+{
+  "admin/editor.video.title": "Video",
+  "admin/editor.video.src.title": "URL",
+  "admin/editor.video.width.title": "Width",
+  "admin/editor.video.height.title": "Height",
+  "admin/editor.video.autoplay.title": "Play automatically",
+  "admin/editor.video.loop.title": "Play in loop",
+  "admin/editor.video.poster.title": "Poster URL",
```

I know poster is the API term, but I think the term thumbnail is more used
```suggestion
  "admin/editor.video.poster.title": "Thumbnail image URL",
```